sessions/june-2018/2-mynavigation/css/style.css

```diff
@@ -0,0 +1,46 @@
+body{
+    font-size: 100%;
+    background-color: #e6e7d7;
+}
+
+header {
+    height: 3.75rem;
+    background: #2b2b27;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
Don't use custom system fonts. They are not available everyone
s system. Instead use web fonts.